docs/resources/s3/bucket_server_side_encryption_configuration.md

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
having separate sub-folders(s3) in data-sources or resources folder might make registry not parse correctly the docs. We should check that it is according to what they expect for structure.